_css/foundation/_foundation.scss

```diff
@@ -65,6 +65,8 @@
 @include foundation-orbit;
 @include foundation-pagination;
 @include foundation-progress-bar;
+@include foundation-prototype-sizing;
```

Comment:
Was this on purpose?

Reply:
@albertvolkman yes, I'm using it already on other task so even if you leave it or not, it is added in another PR... and that is useful for sizing different things, even if I'm not using it on elements on this particular page.

Reply:
Allow the other PR to add it then, please. Remove it here.

Reply:
@albertvolkman I removed it.